src/ctelnet.cpp

```diff
@@ -1425,6 +1425,9 @@ void cTelnet::processTelnetCommand(const std::string& command)
             if (command.size() < 6) {
                 return;
             }
+
+            _m = _m.replace(TN_BELL, QLatin1String("\\\\7"));
```

Comment:
Should leading '_' on members/variables be being used here - I thought it is not considered to be a good thing or very wise: https://stackoverflow.com/questions/228783/what-are-the-rules-about-using-an-underscore-in-a-c-identifier ...

Reply:
That's outside of the scope of the PR review - this code was here before. Happy to change it in this case.